TESTS/mbed_hal/common_tickers/main.cpp

```diff
@@ -37,8 +37,10 @@ extern "C" {
 #define TICKER_INT_VAL 500
 #define TICKER_DELTA 10
 
-#define LP_TICKER_OVERFLOW_DELTA 0 // this will allow to detect that ticker counter rollovers to 0
-#define US_TICKER_OVERFLOW_DELTA 50
+#define LP_TICKER_OVERFLOW_DELTA1   0   // this will allow to detect that ticker counter rollovers to 0
+#define LP_TICKER_OVERFLOW_DELTA2   0
+#define US_TICKER_OVERFLOW_DELTA1   50
+#define US_TICKER_OVERFLOW_DELTA2   60
```

Comment:
It will be good to name the defines as min/max instead 1/2 to have better understanding

Reply:
@deepikabhavnani Because this PR has merged, I would make no modification for it.

Reply:
No problem